# Creating Twin Data Sets
Twin data sets can be generated by simulating reads from the *Asterochloris sp.* and *Cladonia grayi* pseudogenomes using *ART*. The non-normal insert size distribution, as generated by fitting a censored Weibull distribution to real data, can be found in *reads_per_insertsize.txt*. The scripts as given here will generate a 8:2 coverage ratio data set as a simple example. 

## Creating the *ART* commands & running them
Using *00a-create_cladonia8.awk* (and *00b-create_asterochloris2.awk* respectively) will generate a shell script (*01a-run_cladonia8.sh*) that you can use to submit all necessary *ART* commands for a single coverage/organism to the SGE. After those jobs are finished you can use *02a-merge-simulations-cladonia.sh* to uniquely merge the ~400 individual fastq-files that were generated by *ART*. After this you can run *03-flash.sh* to merge overlapping read pairs.

## Workflow
*00a-create_cladonia8.awk* -> *01a-run_cladonia8.sh* -> *02a-merge-simulations-cladonia.sh* -> *00b-create-asterochloris2.awk* -> *01b-run_asterochloris2.sh* -> *02b-merge-simulations-asterochloris.sh* -> *03-flash.sh*

## Results
If everything worked you will have 5 files of interest: 
- The original read pairs (each read will be prefixed with *Astpho* or *Clagr* depending on origin)
 - clad8_ast2-F.fastq
 - clad8_ast2-R.fastq
- The reads after merging 
 - clad8_ast2.extendedFrags.fastq
 - clad8_ast2.notCombined_1.fastq
 - clad8_ast2.notCombined_2.fastq
